### PR TITLE
Sankey diagram: IE test fixes

### PR DESCRIPTION
**Pull request by sokolov11**
Tests for IE fixed

### CODE REVIEW

File: js/viz/sankey/graph.js [hunk 1]
```diff
@@ -4,6 +4,16 @@ const WHITE = 'white';
 const GRAY = 'gray';
 const BLACK = 'black';
 
+let findIndex = function(array, callback) {
```
Comment (roman-simionov) [inline js/viz/sankey/graph.js]: Use inArray method of core/utils/array module instead of this implementation

File: js/viz/sankey/layout.js [hunk 2]
```diff
@@ -5,11 +5,20 @@ const _SPLINE_TENSION = 0.3;
 const graphModule = require('./graph');
 const validatorModule = require('./data_validator');
 
+let find = function(array, callback) {
+    for(let i = 0; i < array.length; i++) {
+        if(callback(array[i])) {
+            return array[i];
+        }
+    }
+    return null;
```
Comment (roman-simionov) [inline js/viz/sankey/layout.js]: Native method returns undefined if it doesn't find an element

File: js/viz/sankey/layout.js [hunk 3]
```diff
@@ -5,11 +5,20 @@ const _SPLINE_TENSION = 0.3;
 const graphModule = require('./graph');
 const validatorModule = require('./data_validator');
 
+let find = function(array, callback) {
```
Comment (roman-simionov) [inline js/viz/sankey/layout.js]: Add this implementation into core/utils/array module

**Review by roman-simionov — CHANGES_REQUESTED** [review]
see my comments in the code

**Review by roman-simionov — DISMISSED** [review]

**Review by pavelgruba — CHANGES_REQUESTED** [review]
We do not need these utils methods

**Review by pavelgruba — APPROVED** [review]

**Pull request merged**